changelog/unreleased/issue-2942

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix nil check in rejectBySize
```

Comment:
Please change to: `Bugfix: Make --exclude-larger-than handle disappearing files`.